src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -300,6 +300,7 @@ public static FeedbackResponseCommentSearchResultBundle fromResults(
                 frcDb.deleteDocument(comment);
                 continue;
             }
+            bundle.sessionTimeZone = session.getTimeZone();
```

Comment:
The time zone of the bundle would be the time zone of the session of the last document in the filtered results. If there are multiple sessions in the filtered result, the time zone of certain comment rows may be wrong.